### ISSUE TITLE
Dialog overlay is not transparent in high contrast 

### ISSUE THREAD

**Issue by shaikshavalis** [microsoft/fluentui#5078]
### Bug Report

- __Package version(s)__: 5.106.1000
- __Browser and OS versions__:  Edge
- __Microsoft Alias__:  v-shsha7

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix?  No

Requested priority: (Blocking, High, Normal, Low) High

Products/sites affected: (if applicable) VSTS

#### Describe the issue:
When the dialog is opened we are not able to see the web page data behind it due to dark overlay

![image](https://user-images.githubusercontent.com/9265381/40919888-8b88088c-6828-11e8-858a-60922f3bb166.png)

#### Actual behavior:

entire page is becoming dark and can only see dialog content not the page behind it as in attached image

#### Expected behavior:

should be able to see the data behind the dialog

### If applicable, please provide a codepen repro:

https://developer.microsoft.com/en-us/fabric#/components/dialog

**Comment by betrue-final-final** [comment]
This may actually be in the overlay component. @lynamemi, seems like a pretty easy fix.

**Comment by lynamemi** [comment]
I'm not able to repro this locally or on the deployed website. Can you tell us more about the circumstances in which you see this bug? Does it happen reliably for you?

**Comment by shaikshavalis** [comment]
I am able to repro this on the office fabric website itself. Here are the repro steps.

1. Go to this page https://developer.microsoft.com/en-us/fabric#/components/dialog
2.  change high contrast setting theme to High contrast Black
3. Click on Open dialog button , then the dialog will open and the data behind the dialog is not visible.

**Comment by betrue-final-final** [comment]
@lynamemi, did you check this in Chrome?

**Comment by lynamemi** [comment]
I definitely still don't repro this in either Edge or Chrome on the fabric website's dialog example, however, I do repro it on Firefox. Firefox seems to not like the decimal alpha.

**Comment by betrue-final-final** [comment]
@lynamemi , did you get back to this yet?
